### ISSUE TITLE
Tainted Magic update

### ISSUE THREAD

**Issue by bartimaeusnek** [GTNewHorizons/GT-New-Horizons-Modpack#2948]
#### Which modpack version are you using?
2.0.4.1 DEV

#
#### What do you suggest instead/what changes do you propose?
We use Tainted Magic 7.6, but there is 7.82 and a even more bug fixed version on the [git](https://github.com/yorkeJohn/Tainted-Magic), if we dont use a custom version, we should maybe update.

**Comment by Dream-Master** [comment]
this will cost a bit work to patch all recipes i made in scripts. Sadly researches was removed and shuffled.

**Comment by Dream-Master** [comment]
not for 2.0.4 1 but for a later update

**Comment by Chaoschaot232** [comment]
If this is the main thread for it I would prefere switching to TM 8 which was released back in Feb 2022.

While I disabled the Script for TM I noticed some quest errors:

![image](https://user-images.githubusercontent.com/1703441/167297797-b0efc9bb-5b60-409e-960d-b568af7a593c.png)

Removed Focis are:

Meteorology, Tainted Storm, Tainted Schockwave, Dark Matter and Time

Some of them got replacements but not as foci, they are now new forms of Salis Mundus and called `Salis Tempestas` (repl. for Meteorology Foci) and `Salis Aevum` (repl. of Time Foci). Still doing the same thing but now one time usage items.

Found also out that Avaritia.zs uses some TM stuff but this is easily replacable with there new unlocalized names which means replacing the gone focis with there new salis alternatives should be fine here too, one example:

![image](https://user-images.githubusercontent.com/1703441/167297978-cb704407-57cc-48ab-9271-c279236521e9.png)

It kinda looks a bit ugly now that it is a shiny yellow dust but it's effect is the same like the foci before.

After all, like Dream mentioned, all the recipes must be looked up again and the mentioned quests have to be redone to fit in again.

Also linking #9048 as it contains images what is going to be removed. At last I would look to maybe rename some unlocalized items to not screw up existing builds to much, if possible.

Leaving this comment here as little reminder on that and which parts are affected from what I have seen while checking out the changes within the current dev pack :)
